### PR TITLE
Pattyssilveira/accordion component

### PR DESCRIPTION
**Pull request by pattyssilveira**
This pt adds:
- Accordion section component
- Accordion component
- Accordion Storybook

### CODE REVIEW

File: components/atoms/accordion-section/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    const iconClass = classnames(styles.downIcon, {
+      [styles.exitIcon]: isOpen,
+    })
+    const title = classnames(styles.title, {
+      [styles.selected]: isOpen,
+    })
+
+    return (
+      <div className={styles.section}>
+        <button className={styles.header} onClick={this.handleClick}>
```
Comment (beatriz1304) [inline components/atoms/accordion-section/index.js]: Change button to a component 
We've already implemented this

File: components/atoms/accordion-section/styles.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
+.section {
+    background-color: white;
```
Comment (beatriz1304) [inline components/atoms/accordion-section/styles.css]: Use color as a variable

File: components/atoms/accordion-section/styles.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+.title {
+    display: flex;
+    width: auto;
+    justify-content: space-between;
+    font-size: 13px;
+    padding: 5px;
+    font-family: var(--font-default);
+    font-weight: 400;
+    color: #979797;
```
Comment (beatriz1304) [inline components/atoms/accordion-section/styles.css]: Same here

File: components/atoms/accordion-section/styles.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    margin: 0;
+}
+
+.selected {
+   color: #4c4c4c;
```
Comment (beatriz1304) [inline components/atoms/accordion-section/styles.css]: Same here

File: components/atoms/accordion-section/styles.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.body {
+    position: relative;
+    transition: height 0.5s;
+    background-color: #f5f6f6;
```
Comment (beatriz1304) [inline components/atoms/accordion-section/styles.css]: Use all colors as a variable

File: components/atoms/accordion-section/index.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        </button>
+        <div className={styles.body} ref={this.bodyRef} style={{ height }}>
+          <p>{this.props.children}</p>
+        </div>
+      </div>
+    )
+  }
+}
+
+Section.propTypes = {
```
Comment (beatriz1304) [inline components/atoms/accordion-section/index.js]: I think you should put this code as a static property inside class

`static propTypes = {}`

File: components/atoms/accordion-section/index.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  onClick: PropTypes.func,
+  className: PropTypes.string,
+  children: PropTypes.node.isRequired,
+  sectionTitle: PropTypes.node,
+  isOpen: PropTypes.bool,
+}
+
+Section.defaultProps = {
```
Comment (beatriz1304) [inline components/atoms/accordion-section/index.js]: Same with the default props

`static defaultProps = {
    isOpen: true,
    onClick: () => {},
     ...
  }`

File: components/atoms/accordion-section/styles.css [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+    margin: 0;
+}
+
+.selected {
+   color: #4c4c4c;
+}
+
+.body {
+    position: relative;
+    transition: height 0.5s;
```
Comment (beatriz1304) [inline components/atoms/accordion-section/styles.css]: Just for information, it's possible to use `.5s` without 0

File: components/molecules/accordion/index.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+import styles from './styles.css'
+
+const Accordion = ({ children }) => <div className={styles.accordion}>{children}</div>
+
+Accordion.propTypes = {
+  children: PropTypes.node.isRequired,
+}
+
+Accordion.defaultProps = {
+  isOpen: false,
```
Comment (beatriz1304) [inline components/molecules/accordion/index.js]: Remove the unused props

File: components/atoms/accordion-section/index.js [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+      [styles.selected]: isOpen,
+    })
+
+    return (
+      <div className={styles.section}>
+        <button className={styles.header} onClick={this.handleClick}>
+          <h1 className={title}>{this.props.sectionTitle}</h1>
+          <i className={iconClass} />
+        </button>
+        <div className={styles.body} ref={this.bodyRef} style={{ height }}>
```
Comment (beatriz1304) [inline components/atoms/accordion-section/index.js]: I don't know if it's a good idea to have className and style both as props
We can talk more about this with Doug, DLeite and Mibs

File: components/atoms/accordion-section/index.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+    })
+    const title = classnames(styles.title, {
+      [styles.selected]: isOpen,
+    })
+
+    return (
+      <div className={styles.section}>
+        <button className={styles.header} onClick={this.handleClick}>
+          <h1 className={title}>{this.props.sectionTitle}</h1>
+          <i className={iconClass} />
```
Comment (beatriz1304) [inline components/atoms/accordion-section/index.js]: I think that can have more components

File: components/atoms/accordion-section/index.js [hunk 12]
```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+import styles from './styles.css'
+
+class Section extends Component {
```
Comment (dleitee) [inline components/atoms/accordion-section/index.js]: move section to `/accordion/section.js`

File: components/atoms/accordion-section/index.js [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+  state = {
+    isOpen: true,
+    bodyHeight: 'auto',
+  }
+
+  handleClick = () => {
```
Comment (dleitee) [inline components/atoms/accordion-section/index.js]: I have to think about it.

File: components/atoms/accordion-section/index.js [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    const { isOpen } = this.state
+    this.setState({
+      isOpen: !isOpen,
+    })
+  }
+
+  bodyRef = element => {
```
Comment (dleitee) [inline components/atoms/accordion-section/index.js]: I have to think about it.

File: components/atoms/accordion-section/styles.css (hunk 8 above)
Comment (dleitee) [reply components/atoms/accordion-section/styles.css]: we need to create a stylelint as well

File: components/molecules/accordion/index.js [hunk 15]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
+
+const Accordion = ({ children }) => <div className={styles.accordion}>{children}</div>
```
Comment (dleitee) [inline components/molecules/accordion/index.js]: add `Accordion.Section = Section`

File: components/molecules/accordion/index.js [hunk 16]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './styles.css'
```
Comment (dleitee) [inline components/molecules/accordion/index.js]: add `import Section from './section'`

File: components/molecules/accordion/index.stories.js [hunk 17]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import Section from '_components/atoms/accordion-section'
+
+import Accordion from './index'
+
+storiesOf('Accordion', module).add('Accordion', () => (
+  <Accordion>
+    <Section sectionTitle="Neymar"> O ousado chegou.</Section>
```
Comment (dleitee) [inline components/molecules/accordion/index.stories.js]: use `Accordion.Section`

**Review by dleitee — CHANGES_REQUESTED** [review]

**Comment by pattyssilveira** [comment]
@dleitee @beatriz1304 I made some changes, can you review again? =)

**Review by dleitee — APPROVED** [review]

**Pull request merged**